design-documents/graph-ql/coverage/Wishlist.graphqls

```diff
@@ -0,0 +1,77 @@
+type Query {
+    wishlist: WishlistOutput #already implemented
+}
+
+type Mutation {
+    createWishlist: ID @doc(description:"If not exists - creates new wishlist. If exists - returns wishlist ID for customer.")
+    removeWishlist(id: ID!): Boolean
+    addProductsToWishlist(input: AddProductsToWishlistInput): AddProductsToWishlistOutput
+    removeProductsFromWishlist(input: RemoveProductsFromWishlistInput): RemoveProductsFromWishlistOutput
+    updateProductsInWishlist(input: UpdateProductsInWishlistInput): UpdateProductsInWishlistOutput
+}
+
+type Wishlist {
+    id: ID
+    items: [WishlistItem]
+    items_count: Int
+    sharing_code: String
+    updated_at: String
+}
+
+input AddProductsToWishlistInput {
+    wishlist_id: ID!
+    wishlist_items: [WishlistItemInput!]!
+}
+
+input RemoveProductsFromWishlistInput {
+    wishlist_id: ID!
+    wishlist_items_ids: [ID!]!
+}
+
+input UpdateProductsInWishlistInput {
+    wishlist_id: ID!
+    wishlist_items: [WishlistItemUpdateInput!]!
+}
+
+input WishlistItemUpdateInput {
+    wishlist_item_id: ID
```

Comment:
Let's add `wishlist_id` as well. I know it is not mandatory, but it would be useful for semantics.